test/fulfillment_order_test.rb

```diff
@@ -49,5 +48,80 @@ def setup
         assert_equal 450789469, fulfillment.order_id
       end
     end
+
+    context "#move" do
+      should "be able to move fulfillment order to a new_location_id" do
+        fulfillment_order = ShopifyAPI::FulfillmentOrder.find(519788021)
+        new_location_id = 5
+
+        moved = ActiveSupport::JSON.decode(load_fixture('fulfillment_order'))
+        moved['assigned_location_id'] = new_location_id
+        fulfillment_order.status = 'closed'
+
+        body = {
+            original_fulfillment_order: fulfillment_order,
+            moved_fulfillment_order: moved,
+            remaining_fulfillment_order: nil,
+        }
+        api_version = ShopifyAPI::ApiVersion.find_version('2019-01')
+        endpoint = "fulfillment_orders/519788021/move"
```

Comment:
Looks like this should use `#{moved.id}`

Reply:
The URL paths for REST from routes:
```
    resources :fulfillment_orders, only: [:show] do
        member do
            post :cancel
            post :close
            post :move
        end
```
So the path ends up being .../fulfillment_orders/519788021/move.json